diff -Naur xfwm4-0.0.0-20090419/themes/daloa/Makefile.am xfwm4-0.0.0-20090419.patched/themes/daloa/Makefile.am
--- xfwm4-0.0.0-20090419/themes/daloa/Makefile.am	2009-05-02 08:34:23.000000000 -0400
+++ xfwm4-0.0.0-20090419.patched/themes/daloa/Makefile.am	2009-05-23 09:47:15.000000000 -0400
@@ -14,7 +14,6 @@
 	hide-active.png							\
 	hide-inactive.png						\
 	hide-pressed.png						\
-	hide-pressed.png						\
 	hide-prelight.png						\
 	left-active.png							\
 	left-inactive.png						\
diff -Naur xfwm4-0.0.0-20090419/themes/default/Makefile.am xfwm4-0.0.0-20090419.patched/themes/default/Makefile.am
--- xfwm4-0.0.0-20090419/themes/default/Makefile.am	2009-05-02 08:34:23.000000000 -0400
+++ xfwm4-0.0.0-20090419.patched/themes/default/Makefile.am	2009-05-23 09:50:56.000000000 -0400
@@ -15,7 +15,6 @@
 	hide-inactive.png						\
 	hide-prelight.png						\
 	hide-pressed.png						\
-	hide-pressed.png						\
 	left-active.png							\
 	left-inactive.png						\
 	maximize-active.png						\
diff -Naur xfwm4-0.0.0-20090419/themes/default-4.6/Makefile.am xfwm4-0.0.0-20090419.patched/themes/default-4.6/Makefile.am
--- xfwm4-0.0.0-20090419/themes/default-4.6/Makefile.am	2009-05-02 08:34:24.000000000 -0400
+++ xfwm4-0.0.0-20090419.patched/themes/default-4.6/Makefile.am	2009-05-23 09:52:08.000000000 -0400
@@ -15,7 +15,6 @@
 	hide-inactive.png						\
 	hide-prelight.png						\
 	hide-pressed.png						\
-	hide-pressed.png						\
 	left-active.png							\
 	left-inactive.png						\
 	maximize-active.png						\
diff -Naur xfwm4-0.0.0-20090419/themes/kokodi/Makefile.am xfwm4-0.0.0-20090419.patched/themes/kokodi/Makefile.am
--- xfwm4-0.0.0-20090419/themes/kokodi/Makefile.am	2009-05-02 08:34:23.000000000 -0400
+++ xfwm4-0.0.0-20090419.patched/themes/kokodi/Makefile.am	2009-05-23 09:53:18.000000000 -0400
@@ -15,7 +15,6 @@
 	hide-inactive.png						\
 	hide-prelight.png						\
 	hide-pressed.png						\
-	hide-pressed.png						\
 	left-active.png							\
 	left-inactive.png						\
 	maximize-active.png						\
